R/extract.R

```diff
@@ -119,7 +119,7 @@ extract_parameter_samples <- function(stan_fit, data, reported_dates, reported_i
   out$growth_rate <- extract_parameter(
     "r",
     samples,
-    reported_dates
+    reported_dates[-1]
```

Comment:
Instead of dropping the first time step, update `calculate_growth` to either get an approximate value from `Rt` or set it to `NA` .